{
  RemoteServiceServletTestServiceAsync service=getAsyncService();
  delayTestFinishForRpc();
  service.throwUnknownRuntimeException(new AsyncCallback<Void>(){
    @Override public void onFailure(    Throwable caught){
      assertTrue(caught instanceof InvocationException);
      finishTest();
    }
    @Override public void onSuccess(    Void result){
      fail();
    }
  }
);
}
